src/app/issues/[id]/page.tsx

```diff
@@ -5,19 +5,17 @@ import ViewPointList from "@/components/Conversation/ViewPoints/ViewPointList";
 import type { Metadata } from "next";
 
 type IssueViewProps = {
-    params: {
-        id: string;
-    };
+    params: Promise<{ id: string }>;
```

Comment:
I don't understand why we need to make params a promise object. Is it due to SSR?